packages/flutter_tools/lib/src/test/flutter_web_goldens.dart

```diff
@@ -118,16 +129,17 @@ class TestGoldenComparatorProcess {
         .transform<String>(utf8.decoder)
         .transform<String>(const LineSplitter())
         .forEach((String line) {
-          globals.printError('<<< $line');
+          logger.printError('<<< $line');
         });
   }
 
+  final Logger _logger;
   final Process process;
   StreamIterator<Map<String, dynamic>> streamIterator;
 
   Future<void> close() async {
-    await process.stdin.close();
```

Comment:
I found that with the faked responses It was hard to get stdin to close when i wanted to. I haven't really seem examples where we close stdin like this before, I'm wondering if we should be following that pattern?

Reply:
I don't know what the subprocess is here, but killing and waiting for the exit code is sufficient in most cases.